src/components/Modals/Settings/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { locales } from './constants'
+
+export const getLocaleLabel = (key: string): string => locales.find(l => l.key === key)?.label ?? ''
```

Comment:
similarly this should move closer to the core translations stuff

Reply:
put in a new utils.ts here https://github.com/shapeshift/web/tree/develop/src/assets/translations